pkg/static/login.js

```diff
@@ -785,7 +802,9 @@
                 } else {
                     if (window.console)
                         console.log(xhr.statusText);
-                    if (xhr.statusText.indexOf("authentication-not-supported") > -1) {
+                    if (xhr.statusText.startsWith("captured-stderr:")) {
+                        show_captured_stderr(decodeURIComponent(xhr.statusText.replace(/^captured-stderr:/, '')));
+                    } else if (xhr.statusText.indexOf("authentication-not-supported") > -1) {
```

Comment:
Similarly here -- known error codes should be handled first, and captured stderr should be the fallback.